.project/scripts/new-tool-script/template/tool/package.json

```diff
@@ -15,7 +15,7 @@
 		"vue": "^2.6.11"
 	},
 	"devDependencies": {
-		"@holistic-web/eslint-config-toolbox": "^1.0.5",
+		"@holistic-web/eslint-config": "^0.1.2",
```

Comment:
Why are we going down to 0.1.2 @michaelfitzhavey ?

Reply:
As they are different packages they have been updated different number of times on npm. Our new config is currently on version `0.1.2` https://www.npmjs.com/package/@holistic-web/eslint-config